Formula/p/python@3.12.rb

```diff
@@ -460,23 +464,12 @@ def sitecustomize
   def caveats
     <<~EOS
       Python has been installed as
-        #{HOMEBREW_PREFIX}/bin/python#{version.major_minor}
+        #{HOMEBREW_PREFIX}/bin/python3
 
-      Unversioned and major-versioned symlinks `python`, `python3`, `python-config`, `python3-config`, `pip`, `pip3`, etc. pointing to
-      `python#{version.major_minor}`, `python#{version.major_minor}-config`, `pip#{version.major_minor}` etc., respectively, have been installed into
+      Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
+      `python3`, `python3-config`, `pip3` etc., respectively, have been installed into
         #{opt_libexec}/bin
 
-      You can install Python packages with
-        pip#{version.major_minor} install <package>
-      They will install into the site-package directory
-        #{HOMEBREW_PREFIX}/lib/python#{version.major_minor}/site-packages
-
-      tkinter is no longer included with this formula, but it is available separately:
-        brew install python-tk@#{version.major_minor}
```

Comment:
Did we want to remove this caveat? There is also the `python-gdbm` caveat in `python@3.11` that didn't carry over to `python@3.12`.

`python-tk@3.12` and `python-gdbm@3.12` will remain as they are part of base Python but we just put them into separate formulae (given that Homebrew doesn't support sub/split-packages).

Not important to handle here but may want to add caveats in follow up.

Reply:
I agree that it makes sense to handle this in a followup syntax PR.

Reply:
I removed that line because it now felt useless. We had it for a few years now I think. I hope our users have adjusted their workflows, and my hope was to stop telling obvious things in the caveats.
But if you think we need to add it back: this can be done in a syntax-only follow-up pull request.

Reply:
It doesn't matter to me whether we keep or remove it.

Homebrew's way of separate formulae is a bit odd since we don't support sub/split-packages. Most repositories would either install everything together (e.g. Arch Linux) or use subpackages (which colocates the related packages that use the same source distribution).